src/app/data-client.ts

```diff
@@ -14,8 +14,8 @@ export type FilterOptions = Partial<pb.Filter.AsObject> & {
 };
 
 type TabularData = {
-  data?: googleStructPb.Struct.AsObject;
-  metadataIndex: number;
+  data?: googleStructPb.JavaScriptValue;
```

Comment:
I think this will actually be a `{[key: string]: googleStructPb.JavaScriptValue};` based on what toJavaScript() returns

Reply:
I wonder why it passed as just a regular javascriptvalue type. Is it okay that this was not picked up by lint?

Reply:
that's weird, but maybe just a weird issue